owlbot.py

```diff
@@ -68,35 +69,6 @@
     [".github/header-checker-lint.yml"], '"Google LLC"', '"pandas-gbq Authors"',
 )
 
-# Work around bug in templates https://github.com/googleapis/synthtool/pull/1335
-s.replace(".github/workflows/unittest.yml", "--fail-under=100", "--fail-under=96")
-
-# Add environment variables to build.sh to support conda virtualenv 
-# installations
-s.replace(
-    [".kokoro/build.sh"],
-    "export PYTHONUNBUFFERED=1",
-    r"""export PYTHONUNBUFFERED=1
-export CONDA_EXE=/root/conda/bin/conda
-export CONDA_PREFIX=/root/conda
-export CONDA_PROMPT_MODIFIER=(base) 
-export _CE_CONDA=
-export CONDA_SHLVL=1
-export CONDA_PYTHON_EXE=/root/conda/bin/python
-export CONDA_DEFAULT_ENV=base
-export PATH=/root/conda/bin:/root/conda/condabin:${PATH}
-""",
-)
```

Comment:
I think we need to keep this, right?

Reply:
@parthea

Reply:
I added `.kokoro/build.sh` to the list of excludes so no replacements will be needed for `.kokoro/build.sh` moving forward. With the proposed changes, `.kokoro/build.sh` will be maintained as a handwritten file rather than an autogenerated one.